service-front/module/Application/view/application/pages/cp/confirm_dob.twig

```diff
@@ -21,7 +21,7 @@
             </div>
             <div class="govuk-form-group">
                 <h2 class="govuk-heading-m">Enter date of birth</h2>
-                <p class="govuk-body">Enter date of birth, for example 31 03 1980</p>
+                <p class="govuk-body app-text-gray">Enter date of birth, for example 31 03 1980</p>
```

Comment:
There's a class in the design system for this: `govuk-hint`

Reply:
The design feedback has said that the hint text is in the wrong colour, so I customised it as per Figma